### ISSUE TITLE
Hospitality: Force Visit Debug Option

### ISSUE THREAD

**Issue by TheNewWorkcloxks** [OrionFive/Hospitality#813]
Problem: V.O.I.D. Faction can't visit
Currently, for some odd reason, once you are allied with V.O.I.D. you can't request them for visits. It says you can, you press on it and the dialogue that occurs for all the other factions when you invite them pops up, but then when you are back to the comms console, it doesn't say they are en route, it just says invite for a visit again. 

Primary Option: Add a force visit option
Allows you can choose any (neutral) faction to invite in the debug console. Smilar to the raid command, it would function like this. [Force Visit] -> [Faction] -> [Amount] Would be a neat thing to have. Also very important to make it look like you have a bustling city without having to raise and capture your own pawns. 

Alt Solution: Maybe find the root of the problem
It's a mod compatibility problem so I don't really expect you to make an entire compatibility thing. The primary solution works aswell.

**Comment by OrionFive** [comment]
Yeah, I'm afraid the compatibility will have to come from their end. Other factions work fine, usually.

I suppose the force visit thing could be done. You can already trigger visits from a random faction via the incidents menu in the dev tools. Maybe that's enough for what you need?

**Comment by TheNewWorkcloxks** [comment]
Probably. Like the raid command, it allows you to raid with a faction, would be cool if you could choose which faction visits you. Just an idea though! I also love how responsive and interactive you are with your community!

**Comment by OrionFive** [comment]
Thanks, I try. But truth be told, I don't have the time anymore for new features. I still get around to fix a critical bug now and then or do a game update. Other than that, it's mostly just maintaining this list of issues, without the time and energy to actually do something with it.

When Patreon was still going well I could hire someone to tackle some of the issues now and then, but that's not really viable anymore either.

So thanks for the praise, but it's not really warranted anymore 😔
